.github/workflows/labels.yml

```diff
@@ -0,0 +1,44 @@
+name: Add Label
+
+on:
+  pull_request:
+    types: [ edited, labeled, opened, reopened ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  no_ticket:
+    name: "Label whether ticket linked from title"
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: kaisugi/action-regex-match@v1.0.1
```

Comment:
In the past I've been advised to use a hash for the action instead of a version because the version could be edited to a different hash and could potentially introduce a security issue for the repo.